cmd/state/main.go

```diff
@@ -84,8 +84,9 @@ func main() {
 	// Run our main command logic, which is logic that defers to the error handling logic below
 	err = run(os.Args, isInteractive, out)
 	if err != nil {
+		silent := isSilent(err)
 		exitCode, err = unwrapError(err)
-		if !isSilent(err) {
+		if !silent {
```

Comment:
```suggestion
		if err != nil {
```

Looking at the code of `unwrapError`, this feels more appropriate.